Source/SteveSloth/Public/MyEnemyAttackingState.h

```diff
@@ -0,0 +1,33 @@
+/****************************************************************************************
+ * Copyright: SteveSloth
+ * Name: Ken Ferris
+ * Script: MyEnemyAttackingState.h
+ * Date: May 3, 2024
+ * Description: Enemy Attack State
+ * TODO:
+ * Known Bugs:
+ ****************************************************************************************/
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "MyEnemyBaseState.h"
+#include "MyEnemyAttackingState.generated.h"
+
+/**
```

Comment:
these empty comments should not be here.. please clean up the code and indent properly